libiocage/lib/JsonConfigFile.py

```diff
@@ -0,0 +1,62 @@
+import os.path
+import json
+
+import libiocage.lib.helpers
+
+
+class JsonConfigFile:
+
+    def __init__(
+        self,
+        file: str,
+        logger: libiocage.lib.Logger.Logger
+    ):
+
+        libiocage.lib.helpers.init_logger(self, logger)
+        self._file = file
+
+    @property
+    def file(self):
+        return self._file
+
+    def read(self) -> dict:
+        try:
+            with open(self.file, "r") as conf:
+                return json.load(conf)
+        except:
```

Comment:
We should try not to be so wide with our exceptions. Explicit is better.

Reply:
Removed the exception at all. Either a file exists and is loaded or not. We still want the exception to occur, when the JSON data is invalid.